_opensearch/install/compatibility.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Compatibility
+parent: Install OpenSearch
+nav_order: 2
+---
+
+# Operating system and JVM compatibility
+
+- We recommend installing OpenSearch on RHEL- or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, and Ubuntu (LTS). OpenSearch should work on many Linux distributions, but we only test a handful.
+- OpenSearch ships with a compatible version of Java in the `jdk` directory. To find its version, run `./bin/opensearch --version`.
+
+  To use a different Java installation, set the `OPENSEARCH_JAVA_HOME` environment variable to the Java install location. We recommend Java 11 (LTS), but OpenSearch also works with Java 8.
```

Comment:
It looks like our gradle build can test on a wide variety of Java versions (8-12?), but my understanding is that we only want people using the LTS ones for self-installs (8, 11, eventually 17). Again, let me know if I've got it wrong here.